src/Microcks.Testcontainers/Model/Message.cs

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright The Microcks Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//
+
+using System.Text.Json.Serialization;
+
+namespace Microcks.Testcontainers.Model;
+
+/// <summary>
+/// Domain object representing a microservice operation / action invocation response.
+/// </summary>
+public class Message
```

Comment:
Use the abstract modifier in a class declaration to indicate that a class is intended only to be a base class of other classes, not instantiated on its own. 

```suggestion
public abstract class Message
```

Reply:
Resolved in new commit